### ISSUE TITLE
Problem with missing values in loci2genind

### ISSUE THREAD

**Issue by mstetter** [emmanuelparadis/pegas#20]
Hi,

there seems to be a newly introduced problem with `. /.` data. I tried to reanalyse some data with a script for about 6 months ago and after reading the data from a vcf file with `read.vcf` I would like to convert it to a genind object with loci2genind and I get the following error

> Error in .local(.Object, ...) : 
>   more than one '.' in column names; please name column as [LOCUS].[ALLELE]

It does not happen when I use a file without missing values. I guess it's some problem with the `./.` but I can't figure out what happened.

Would be awesome if you could help me solving this issue.

Thanks a lot

Markus

**Comment by thibautjombart** [comment]
This is because "." cannot be used to code characters in `genind` objects; current devel version of adegenet will partly sort out the problem by replacing all "." to "_", but then these are not treated as NAs. @emmanuelparadis this should work:

```
loci2genind2 <- function (x) 
{
    ipop <- which(names(x) == "population")
    pop <- if (length(ipop)) 
        x[, ipop]
    else NULL
    df2genind(as.matrix(x[, attr(x, "locicol")]), sep = "/", 
        pop = pop, NA.char = "_")
}
```

Note that `sep` should not be a regular expression.

**Comment by emmanuelparadis** [comment]
Thanks! Fixed in pegas 0.9-0.2 (now on GH).

About `sep`, genotypes in pegas can be phased and the alleles are separated with `|`, hence the regexp. Maybe it'd be good to have a function `unphase` to turn phased genotypes into unphased ones.

**Comment by emmanuelparadis** [comment]
`unphase` is now in pegas on GH. `loci2genind` has now an option `unphase = TRUE`.
